### PR TITLE
Correct typos in "Active Record Query Interface" guide

### PR DESCRIPTION
**Pull request by laurmurclar**
### Summary

Added some missing #'s for comments in code snippets.

### CODE REVIEW

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -1660,9 +1660,9 @@ Client.find_by_sql("SELECT * FROM clients
 ```ruby
 Client.connection.select_all("SELECT first_name, created_at FROM clients WHERE id = '1'")
 # => [
-  {"first_name"=>"Rafael", "created_at"=>"2012-11-10 23:23:45.281189"},
-  {"first_name"=>"Eileen", "created_at"=>"2013-12-09 11:22:35.221282"}
-]
+  # {"first_name"=>"Rafael", "created_at"=>"2012-11-10 23:23:45.281189"},
+  # {"first_name"=>"Eileen", "created_at"=>"2013-12-09 11:22:35.221282"}
+# ]
```
Comment (vipulnsward) [inline guides/source/active_record_querying.md]: Can you check the indentation on these?

**Comment by vipulnsward** [comment]
@laurmurclar thanks for change. Can you check indentation on all the changes?
Also please add [ci skip] to the commit message, so that we don't run tests on Travis, since this is a doc-only change.

**Comment by laurmurclar** [comment]
Absolutely, have committed those changes. Forgot to add the [ci skip], sorry! 😥

**Comment by vipulnsward** [comment]
@laurmurclar can you squash the commits as well?

**Comment by laurmurclar** [comment]
Done, but I'm not sure why that Travis CI build failed (I shouldn't have ran it anyway).

**Comment by vipulnsward** [comment]
Thanks @laurmurclar . 

Please remember to add ` [ci skip]` in the commit message when doing doc-only changes, as I mentioned before. That will avoid the Travis run.

**Comment by laurmurclar** [comment]
Will do, just kept forgetting! Sorry about that.

**Pull request merged**